docs/man/sat-status.8.rst

```diff
@@ -71,6 +71,10 @@ These options must be specified after the subcommand.
         session template itself, and these changes would not be reflected in
         any BOS running or completed session either.
 
+**--bos-version BOS_VERSION**
```

Comment:
Should also add `--bos-fields` while you're at it.

Isn't it the case that `--bos-fields` requires `--base-version v2`? It would make sense to document that in the doc for `--bos-fields`, IMO.

Reply:
Yes, that's correct. The man page has been updated to reflect that.